_drafts/2017-09-01-first-review-experiences.md

```diff
@@ -0,0 +1,137 @@
+---
+name: first-review-experiences
+layout: post_discourse
+title: Experiences as a first time rOpenSci package reviewer
+date: 2017-09-01
+authors:
+  - name: Verena Haunschmid
+    url: http://blog.haunschmid.name
+tags:
+- R
+- software
+- review
+- ropensci
+- onboarding
+---
+
+It all started January 26^th^ this year when I signed up to volunteer as
+a reviewer for R packages submitted to rOpenSci. My main motivation for
+wanting to do package reviews was to learn something new and to
+contribute to the R open source community.
+
+Three months later I was contacted by [Ma�lle Salmon](https://twitter.com/ma\_salmon) asking whether I was interested in
+reviewing the R package ['patentsview'](https://github.com/ropensci/patentsview) for rOpenSci. And yes, I
+was! To be honest I was a little bit thrilled.
+
+The packages are submitted for review to rOpenSci via an issue to their
+github repository and also the reviews happen there. So you can check out all
+previous package submissions and reviews. With all the information you
```

Comment:
Consider adding this link: 
[all previous package submissions and reviews](https://github.com/ropensci/onboarding/issues)